### PR TITLE
Fix Http Status Code with Otel Bridge

### PR DESCRIPTION
**Pull request by alanprot**
HttpStatus code is not working correctly as status code is set as int16 but the otel bridge does not expect int16 attributes. As result, the status code is set as string.

See: https://github.com/opentracing-contrib/go-stdlib/blob/master/nethttp/server.go#L143-L145

This is problematic as the exporters (xray for instance)  expects the status code to be set as an int and defaults to `0` if its not.

See: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6dc71bd739b4eaa407a9a596dc57ca4884e6313e/exporter/awsxrayexporter/internal/translator/http.go#L57-L59

Before:
Screen Shot 2022-10-06 at 9 04 11 PM

After: 

Screen Shot 2022-10-06 at 9 02 46 PM

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]
Please add a changelog entry.

**Comment by alanprot** [comment]
> Please add a changelog entry.

done! thx!

**Review by dmathieu — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

**Comment by alanprot** [comment]
Hi.. 

Is anything missing to merge this pr ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Upgrade `golang.org/x/sys/unix` from `v0.0.0-20210423185535-09eb48e85fd7` to `v0.0.0-20220919091848-fb04ddd9f9c8`.
   This addresses [GO-2022-0493](https://pkg.go.dev/vuln/GO-2022-0493). (#3235)
 - Update histogram default bounds to match the requirements of the latest specification. (#3222)
+- Fix Http Status Code in OpenTracing bridge (`go.opentelemetry.io/otel/bridge/opentracing`).  (#3265)
```
Comment (MrAlias) [inline CHANGELOG.md:22]:
```suggestion
- Encode the HTTP status code in the OpenTracing bridge (`go.opentelemetry.io/otel/bridge/opentracing`) as an integer.  (#3265)
```

**Pull request merged**